### DISCUSSION TITLE
default password or reinitialisation

### DISCUSSION THREAD

**Discussion by mattnemoz** [awawa-dev/HyperHDR#244] [Q&A] [answered]
Hello !
I change the password, but I can't login with it. I don't know why,..
I'm sure of it, but still error.
How can i change ou reinitialise the password please ?
Matt

**Comment by awawa-dev** [comment, marked as answer]
Hi
Just run from command line: ```hyperhdr -resetPassword```

**Reply by mattnemoz** [reply]
Thank you ! :-)
I tried, and had a message that the password was reset (hyperhdr), but doesn't work. 
It worked with the password that I think I had before... 
Very strange

**Reply by awawa-dev** [reply]
Yes, indeed. Maybe something cached in the browser (cookies / password) or it's a bug (but I need a repeating path to reproduce such a problem in my setup, I often use setting password and it works in my case).
